src/cephadm/cephadm

```diff
@@ -1730,10 +1764,6 @@ def call(ctx: CephadmContext,
         async for line in reader:
             message = line.decode('utf-8')
             collected.write(message)
-            if verbosity == CallVerbosity.VERBOSE:
-                logger.info(prefix + message.rstrip())
-            elif verbosity != CallVerbosity.SILENT:
-                logger.debug(prefix + message.rstrip())
```

Comment:
It seems like we are removing logging on these cases. Just to make sure we are doing it on purpose.

Reply:
I think this should all still get logged. In the old code, it was directly making log calls from this internal function `tee` with the stdout/stderr and then we were only doing additional logging afterwards if there was an error. Now it calls `_log_from_call` function with the stdout and stderr after the fact always instead of doing it within `tee`